source/community/reactnative/src/components/actions/select-Action.js

```diff
@@ -32,7 +32,8 @@ export class SelectAction extends React.Component {
 	onClickHandle() {
 		switch (this.payload.type) {
 			case Constants.ActionSubmit:
-				let actionObject = { "type": Constants.ActionSubmit, "data": this.props.selectActionData.data };
+				const title = this.props.selectActionData.title
```

Comment:
Shall we read the properties by destructuring the payload for better readability.

**FROM**
```
const title = this.props.selectActionData.title
let actionObject = { "type": Constants.ActionSubmit, "title":Utils.isNullOrEmpty(title) ? "" : title, "data": this.props.selectActionData.data };
this.onExecuteAction(actionObject);
```
**TO**
```
const { type, title = "", data } = selectActionData;
this.onExecuteAction({type, title, data})
```

Reply:
Resolved